openpype/hosts/blender/plugins/publish/extract_blend.py

```diff
@@ -192,22 +221,35 @@ def _process_resources(
 
             # Get source and destination paths
             sourcepath = image.filepath  # Don't every modify source_image
```

Comment:
We are almost always using it with `Path`, so convert it here.
```suggestion
            sourcepath = Path(image.filepath)  # Don't every modify source_image
```